### PR TITLE
[preview] update to gen108 VM image

### PR DESCRIPTION
**Pull request by kylos101**
## Description

Update to the latest image. 

## Related Issue(s)

Fixes ENG-1668

## How to test

Start a workspace in preview:
image from https://github.com/kylos101/gitpod-custom-image

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - kylos101-gd9f4c157e2</li>
	<li><b>🔗 URL</b> - <a href="https://kylos101-gd9f4c157e2.preview.gitpod-dev.com/workspaces" target="_blank">kylos101-gd9f4c157e2.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - kylos101-gen108-preview-gha.22896</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-kylos101-gd9f4c157e2%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-core-dev" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: dev/preview/infrastructure/scripts/bootstrap-k3s.sh [hunk 1]
```diff
@@ -6,15 +6,19 @@ set -eo pipefail
...
 /usr/local/bin/install-k3s.sh \
   --token "1234" \
-  --node-ip "$(hostname -I | cut -d ' ' -f1)" \
+  --node-ip "$SERVICE_DNS_IP" \
   --node-label "cloud.google.com/gke-nodepool=control-plane-pool" \
   --container-runtime-endpoint=/var/run/containerd/containerd.sock \
   --write-kubeconfig-mode 444 \
   --disable traefik \
   --disable metrics-server \
+  --disable-network-policy \
```
Comment (kylos101) [inline dev/preview/infrastructure/scripts/bootstrap-k3s.sh:20]: `--disable-network-policy \` is recommended by k3s and was required to fix, prior to which k3s failed because the CNI wasn't ready, so we never tried to load calico manifests later, as this bootstrapping script failed here.

File: dev/preview/infrastructure/scripts/bootstrap-k3s.sh [hunk 2]
```diff
@@ -6,15 +6,19 @@ set -eo pipefail
...
+  --disable-cloud-controller \
```
Comment (Furisto) [inline dev/preview/infrastructure/scripts/bootstrap-k3s.sh:21]: Why disable cloud-controller?

Comment (Furisto) [reply dev/preview/infrastructure/scripts/bootstrap-k3s.sh:21]: Ah, got it. Prod parity

**Comment by Furisto** [comment]
Workspaces start successfully

File: dev/preview/infrastructure/modules/gce/cloudinit.yaml [hunk 3]
```diff
@@ -23,6 +23,6 @@ write_files:
         password = "${dockerhub_passwd}"
       EOF
 
-      sudo systemctl restart containerd.service &
+      sudo systemctl restart containerd.service --wait
```
Comment (Furisto) [inline dev/preview/infrastructure/modules/gce/cloudinit.yaml:26]: Why was this added?

**Review by kylos101 — COMMENTED** [review]
Hey @Furisto , here's some feedback, sorry , I meant to share a self-review, but, forgot to click submit.

File: dev/preview/infrastructure/modules/gce/cloudinit.yaml (hunk 3 above)
Comment (kylos101) [reply dev/preview/infrastructure/modules/gce/cloudinit.yaml]: Fixed in 7ff032e540cc479fadc58651213f627837a66c06

Comment (kylos101) [reply dev/preview/infrastructure/modules/gce/cloudinit.yaml]: image

**Review by Furisto — APPROVED** [review]
